src/drag.js

```diff
@@ -0,0 +1,61 @@
+import './style.css';
+import { list } from './index.js';
+
```

Comment:
- [ ] it's best to use a single starting point and import everything there, this can give two benefits
  1. If you ever add a new feature, you'd just import it to the same place, instead of having it to import it to everything else
  2. Control when to call a certain funtion

so it's best to import this to `index.js` and not the other way around.